.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+name: CI
+on: push
```

この戦略は考えたいです。例えば、tagがpushされたタイミングのみリリースを実行するのはどうですか？

```suggestion
on:
  push:
    tags:
      - "*"
```

https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs

---

タグのpushがあったときのみ実行するように変更しました。